docs/api-auth.md

```diff
@@ -25,30 +27,123 @@ There are three different ways a user is considered an admin:
+1. Select `Users and groups`
 
-AWS also provides [examples for a number of languages in their docs](https://docs.aws.amazon.com/general/latest/gr/signature-v4-examples.html).
+    ![Users and Groups](./img/usersandgroups.png)
+
+1. Create a user
+
+    ![Create User](./img/createuser.png)
+
+1. Name the user and provide a temporary password. You may uncheck all of the boxes and leave the other fields blank. This user will not have admin priviliges.
+
+    ![Quick start user](./img/quickstartuser.png)
+
+1. Create a second user to serve as a system admin. Follow the same steps as you did for creating the first user, but name this one something appropriate for their role as an administrator.
+
+    ![Quick start admin](./img/quickstartadmin.png)
+
+1. Create a group
+
+    ![Create group](./img/creategroup.png)
+
+1. Users in this group will be granted admin access to DCE. The group name must contain the term `Admin`. Choose a name and click on the `Create group` button.
+
+    ![Group name](./img/groupname.png)
+
+1. Add your admin user to the admin group to grant them admin privileges.
+    ![Quick start admin detail](./img/quickstartadmindetail.png)
+    ![Add to group](./img/addtogroup.png)
+
+1. Type `dce auth` in your command terminal. This will open a browser with a log in screen. Enter the username and password for the non-admin user that you created. Reset the password as prompted.
+
+    ![Quick start user login](./img/quickstartuserlogin.png)
+
+1. Upon successfully logging in, you will be redirected to a credentials page containing a temporary authentication code. Click the button to copy the auth code to your clipboard.
+
+    ![Credentials Page](./img/credspage.png)
 
+1. Return to your command terminal and paste the auth code into the prompt.
 
+    ```
+    dce auth
+    ✔ Enter API Token: : █
+    ```
 
-### IAM Policy for DCE API requests
+1. You are now authenticated as a DCE User. Test that you have proper authorization by typing `dce leases list`.
+This will return an empty list indicating that there are currently no leases which you can view.
+If you are not properly authenticated as a user, you will see a permissions error.
 
-The IAM principal used to send requests to the DCE API must have sufficient permissions to execute API requests.
+    ```
+    dce leases list
+    []
+    ```
 
-The Terraform module in the repo provides an IAM policy with appropriate permissions for executing DCE API requests. The policy name and ARN are available as Terraform outputs.
+1. Users are not authorized to list child accounts in the accounts pool. Type `dce accounts list` to verify that you get a permissions error when trying to
+view information you do not have access to.
 
+    ```
+    dce accounts list
+    err:  [GET /accounts][403] getAccountsForbidden
+    ```
+
+1. You will need to be authenticated as an admin before continuing to the next section. Type `dce auth` to log in as a different user. Sign out, then enter the username
```

Comment:
This is a little confusing to me to have this in here.
I get you're trying to demonstrate the different auth levels, but it makes it seem like it required to switch roles as part of setting up cognito. I would cut it.